src/app/auth.ts

```diff
@@ -38,7 +36,7 @@ export function initAuth(options:ExplorerOptions, apiService:GraphService, chang
 			// This means no POST operations in <=IE9
 			form: false
 		}
-	});
+	} as any);
```

Comment:
Instead of declaring hello globally as any to accommodate this override, just remove types for this one call.  Note line 13 was removed.  This way, all other hello.js calls in this file still get type checking.